packages/gatsby-cli/src/create-cli.js

```diff
@@ -166,8 +166,10 @@ function buildLocalCommands(cli, isLocalSite) {
     builder: _ =>
       _.option(`prefix-paths`, {
         type: `boolean`,
-        default: false,
-        describe: `Build site with link paths prefixed (set pathPrefix in your gatsby-config.js).`,
+        default: process.env.PREFIX_PATHS || false,
+        describe: process.env.PREFIX_PATHS
+          ? `Build site with link paths prefixed (set by env.PREFIX_PATHS) (set pathPrefix in your gatsby-config.js).`
+          : `Build site with link paths prefixed (set pathPrefix in your gatsby-config.js).`,
```

Comment:
I don't think the help description should change based on this env (plus, how often will you even see this with the env flag set?). Instead I'd use something like 
```
Build site with link paths prefixed. Default is `env.PREFIX_PATHS` or `false`, which is overriden either way by `pathPrefix` in your gatsby-config.js.
```

Will defer to learning for a text there.

Also, it feels weird to me that the gatsby config would override an env var. Shouldn't that be the other way around?

Reply:
I can update this. 

`gatsby-config.js` doesn't override this value though. You can tell `gatsby build` to add path prefixes either by `gatsby build --prefix-paths` or `PREFIX_PATHS=true gatsby build`. In either case though, it will use the pathPrefix (ie, `/some-prefix`) setting from `gatsby-config.js`. If you don't pass the flag or the env variable is fall, paths will not be prefixed, even if the setting is in `gatsby-config.js`.

Reply:
Updated :)